tests/TestCase/TestSuite/FixtureManagerTest.php

```diff
@@ -280,6 +280,63 @@ public function testFixturizeInvalidType()
         $this->manager->fixturize($test);
     }
 
+    /**
+     * Tests handling unmnaged fixtures which don't
+     * import a schema or defined fields. These tables
+     * should only insert records and not drop/create.
+     *
+     * @return void
+     */
+    public function testLoadUnmanagedFixtures()
+    {
+        $schema = new TableSchema('unmanaged', [
+            'title' => ['type' => 'string', 'length' => 100],
+            'body' => ['type' => 'string', 'length' => 100],
+        ]);
+
+        $conn = ConnectionManager::get('test');
+        foreach ($schema->createSql($conn) as $sql) {
+            $conn->execute($sql);
+        }
+        $conn->newQuery()
+            ->insert(['title', 'body'])
+            ->into('unmanaged')
+            ->values([
+                'title' => 'Existing title',
+                'body' => 'Existing body',
+            ])
+            ->execute();
+
+        $buffer = new ConsoleOutput();
```

Comment:
We have an ArrayLog that could be better here if you need to collect logs.

Reply:
I copied that mess from the test. Both should be fixed.